app/_demo.scss

```diff
@@ -0,0 +1,14 @@
+.main-page {
+  Button {
+    font-size: 30;
+    margin:10;
+  }
+}
+.themes {
+  Label {
+    margin: 5;
+  }
+  Button {
+    margin:5;
+  }
```

Comment:
^ those are there just for demo app. Not used in core styles. Just to frame up the demo app 👍